src/main/java/com/zufar/onlinestore/cart/converter/ShoppingSessionItemDtoConverter.java

```diff
@@ -3,23 +3,18 @@
 import com.zufar.onlinestore.cart.dto.ShoppingSessionItemDto;
 import com.zufar.onlinestore.cart.entity.ShoppingSessionItem;
 import com.zufar.onlinestore.product.converter.ProductInfoDtoConverter;
-import com.zufar.onlinestore.product.dto.ProductInfoFullDto;
-import lombok.RequiredArgsConstructor;
-import org.springframework.stereotype.Component;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 
-@Component
-@RequiredArgsConstructor
-public class ShoppingSessionItemDtoConverter {
+@Mapper(uses = ProductInfoDtoConverter.class)
```

Comment:
Does the Mapper annotation create a bean in a Spring context? If not, I suggest using the annotation Component annotation in addition, which will make it easier for us to work and test mapstract converters in our application.